src/AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda/Requests/SqsLambdaAddressChangePostalCodeRequest.cs

```diff
@@ -0,0 +1,29 @@
+namespace AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda.Requests
+{
+    using Abstractions.Requests;
+    using StreetName;
+    using StreetName.Commands;
+
+    public sealed class SqsLambdaAddressChangePostalCodeRequest :
+        SqsLambdaRequest,
+        IHasBackOfficeRequest<AddressBackOfficeChangePostalCodeRequest>,
+        Abstractions.IHasAddressPersistentLocalId
+    {
+        public AddressBackOfficeChangePostalCodeRequest Request { get; set; }
+
+        public int AddressPersistentLocalId { get; set; }
+
+        /// <summary>
+        /// Map to ChangeAddressPostalCode command
+        /// </summary>
+        /// <returns>ChangeAddressPostalCode.</returns>
+        public ChangeAddressPostalCode ToCommand()
+        {
+            return new ChangeAddressPostalCode(
```

Comment:
I think ToCommand should have a PostalCode argument and be called like in the sync handler:
```
            var postInfoIdentifier = request.PostInfoId
                .AsIdentifier()
                .Map(x => x);
            var postalCode = new PostalCode(postInfoIdentifier.Value);

            var cmd = request.ToCommand(
                streetNamePersistentLocalId,
                postalCode,
                CreateFakeProvenance());
```